src/lib/worker.js

```diff
@@ -213,7 +213,9 @@ var invalidReplicaSet = function(db, pods, done) {
 var podElection = function(pods) {
   //Because all the pods are going to be running this code independently, we need a way to consistently find the same
   //node to kick things off, the easiest way to do that is convert their ips into longs and find the highest
-  pods.sort(function(a,b) {
+  pods
+  .filter(function(pod) { return !k8s.podContainsLabels(pod, config.mongoPodLabelCollectionArbiter); })
```

Comment:
I think this function is currently not working as intended. The caller expects the pods array it passes into the function to be sorted in-place, which the filter prohibits by creating a copy. I need to fix that.